daffodil-cli/src/test/resources/org/apache/daffodil/CLI/cli_schema_04.dfdl.xsd

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:dfdl="http://www.ogf.org/dfdl/dfdl-1.0/" targetNamespace="http://example.com"
+  xmlns:ex="http://example.com"> 
+
+  <xs:include schemaLocation="edu/illinois/ncsa/daffodil/xsd/built-in-formats.xsd"/>
+
+  <xs:annotation> 
+    <xs:appinfo source="http://www.ogf.org/dfdl/" >
+      <dfdl:format ref="ex:daffodilTest1" initiator="" terminator="" separator=""
+        separatorSuppressionPolicy="anyEmpty" separatorPosition="infix"
+        alignment="implicit" alignmentUnits="bits" trailingSkip="0"
+        lengthKind="delimited" documentFinalTerminatorCanBeMissing="yes" encoding="ASCII"
+        leadingSkip="0" textTrimKind="none" representation="text" occursCountKind="parsed"
+        textNumberRep="standard" ignoreCase="no" initiatedContent="no" />
+    </xs:appinfo>
+  </xs:annotation>
+    
+    <xs:simpleType name="simpleAssert">
```

Comment:
Do we need a schema this big if we're just verifying that we get the SDW when built-in-formats.xsd is used?

Reply:
I just copied another schema and changed the import--didn't realize is was so large. I'll trim it down to just what is needed.